Fix all acceptance tests

Two acceptance tests fail, we have to fix them:

- `TestAccScalewayDataSourceImage_Filtered`

```bash
--- FAIL: TestAccScalewayDataSourceImage_Filtered (10.01s)
    testing.go:538: Step 0 error: Error refreshing: 1 error(s) occurred:

        * data.scaleway_image.ubuntu: 1 error(s) occurred:

        * data.scaleway_image.ubuntu: data.scaleway_image.ubuntu: The query returned no result. Please refine your query.
```

- `TestAccScalewayBucket`

```bash
--- FAIL: TestAccScalewayBucket (0.02s)
    testing.go:538: Step 0 error: Error applying: 1 error(s) occurred:

        * scaleway_bucket.base: 1 error(s) occurred:

        * scaleway_bucket.base: Only the AMS1 region is supported at the moment
```

--- a/scaleway/data_source_image_test.go
+++ b/scaleway/data_source_image_test.go
@@ -55,7 +55,7 @@ func TestAccScalewayDataSourceImage_Filtered(t *testing.T) {
 				Config: testAccCheckScalewayImageFilterConfig,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckImageID("data.scaleway_image.ubuntu"),
-					resource.TestCheckResourceAttr("data.scaleway_image.ubuntu", "name", "Ubuntu Precise (12.04)"),
+					resource.TestCheckResourceAttr("data.scaleway_image.ubuntu", "name", "Ubuntu Bionic Beaver"),
 					resource.TestCheckResourceAttr("data.scaleway_image.ubuntu", "architecture", "arm"),
 					resource.TestCheckResourceAttr("data.scaleway_image.ubuntu", "public", "true"),
 					resource.TestCheckResourceAttrSet("data.scaleway_image.ubuntu", "organization"),
@@ -105,7 +105,7 @@ data "scaleway_image" "ubuntu" {
 
 const testAccCheckScalewayImageFilterConfig = `
 data "scaleway_image" "ubuntu" {
-  name_filter = "Precise"
+  name_filter = "Bionic"
   architecture = "arm"
 }
 `

